src/goto-programs/goto_contractor.cpp

```diff
@@ -0,0 +1,359 @@
+//
+// Created by Mohannad Aldughaim on 09/01/2022.
+//
+
+#include "goto_contractor.h"
+
+using namespace std;
```

Comment:
Remove this and use the `std` namespace explicitly.

Reply:
done.